lib/src/ast/sass/statement/silent_comment.dart

```diff
@@ -12,11 +13,49 @@ class SilentComment implements Statement {
   /// The text of this comment, including comment characters.
   final String text;
 
+  /// The text of this comment that are marked as a documentation comment
+  /// (leading triple slashes).
+  ///
+  /// Provided with slashes and common leading whitespace removed.
+  String get docComment => _cleanup(text);
+
   final FileSpan span;
 
   SilentComment(this.text, this.span);
 
   T accept<T>(StatementVisitor<T> visitor) => visitor.visitSilentComment(this);
 
+  // Matches '///' followed by some amount of whitespace.
+  final _leadingWhitespace = RegExp('^(\/\/\/[\t ]*)[^\t ]?');
```

Comment:
Generally speaking, I try to parse strings without regular expressions if at all possible in Sass, both because it's more efficient and because it tends to be easier to read and modify. [`StringScanner`](https://pub.dartlang.org/documentation/string_scanner/latest/) is very useful for this sort of thing.

Reply:
Done.